package-lock.json

```diff
@@ -0,0 +1,9107 @@
+{
+  "name": "@fluentui-react-native/repo",
```

Comment:
Delete this file please. Did component-generator create it?

Reply:
Yep

Reply:
@ValentinaKozlova FYI. It seems like component-generator is using NPM instead of yarn, and therefore generating a package-lock.json? It's also adding dependencies to the root package.json. Otherwise, thanks for the awesome FHL project!